### PR TITLE
[#831]: Changed My Tutor to My Team(connects #831)

### PR DESCRIPTION
**Pull request by Pr0chin**

### CODE REVIEW

File: init_docs/Text.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
     "News_Feed": "News Feed",
     "My_Library": "My Library",
     "My_Courses_Progress": "My Courses Progress",
-    "My_Tutors": "My Tutors",
+    "My_Team": "My Team",
```
Comment (dogi) [inline init_docs/Text.json]: please use plural

File: app/MyApp/app/Router.js [hunk 2]
```diff
@@ -1222,7 +1222,7 @@ $(function() {
             wordsOfLibraryTable.push(App.languageDict.attributes.My_Library);
             wordsOfLibraryTable.push(App.languageDict.attributes.My_Courses_Progress);
             wordsOfLibraryTable.push(App.languageDict.attributes.My_Meetups);
-            wordsOfLibraryTable.push(App.languageDict.attributes.My_Tutors);
+            wordsOfLibraryTable.push(App.languageDict.attributes.My_Team);
```
Comment (dogi) [inline app/MyApp/app/Router.js]: please use plural

**Review by dogi — CHANGES_REQUESTED** [review]
@Pr0chin `Teams` not Team ;)

File: app/MyApp/app/Router.js (hunk 2 above)
Comment (Pr0chin) [reply app/MyApp/app/Router.js]: sure @dogi  👍 :D

**Review by lmmrssa — APPROVED** [review]

**Review by leomaxi — APPROVED** [review]

**Pull request closed without merging**